x-pack/plugin/transform/src/test/java/org/elasticsearch/xpack/transform/transforms/ClientTransformIndexerTests.java

```diff
@@ -309,7 +310,7 @@ public void testDisablePit() throws InterruptedException {
         }
         TransformConfig config = configBuilder.build();
 
-        boolean pitEnabled = config.getSettings().getUsePit() == null || config.getSettings().getUsePit();
+        boolean pitEnabled = TransformEffectiveSettings.isPitDisabled(config.getSettings()) == false;
```

Comment:
Did we want both a `TransformEffectiveSettings.isPitDisabled` and `TransformEffectiveSettings.isPitEnabled` so that we don't have to do the `== false` outside of the `TransformEffectiveSettings`?

Reply:
I had this idea too but I dropped it because this test case is the **only** place which uses negation.
All the production code always checks if the PIT is **disabled**.